pages.pt_BR/common/dvc-add.md

```diff
@@ -0,0 +1,20 @@
+# dvc add
+
+> Adiciona um arquivo modificado para o índice.
+> Mais informações: <https://dvc.org/doc/command-reference/add>.
+
+- Adiciona um arquivo para o índice:
+
+`dvc add {{caminho/para/arquivo}}`
+
+- Adiciona um diretório para o índice:
+
+`dvc add {{caminho/para/diretório}}`
```

Comment:
I notice the `diretório` use "ó" which is different from `diretorio` in the template [common-argument.md](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/common-arguments.md). Are both acceptable?

Reply:
It's acceptable, but we should follow the template. Thanks by the notice. :)

Reply:
Changed to follow the template. Thanks!